src/rss-diff.js

```diff
@@ -1,5 +1,25 @@
-function rssNDiff(itemBefore, itemAfter, field) {
-    let delta
+function rssDiff(itemBefore, itemAfter, field) {
+    let beforeMap = {}
+    let out = []
+    let indicies = []
+
+    // Create a map of field to index for first array
+    for (let i = 0; i < itemBefore.length; i++) {
+        beforeMap[itemBefore[i][field]] = i
+    }
+
+    // If element in second array does not exist in previous map, store its index
+    for (let i = 0; i < itemAfter.length; i++) {
+        if (!(itemAfter[i][field] in beforeMap)) {
```

Comment:
I think this is only handling dependency additions. What if a dependency was deleted? It would not show up in the second array, therefore you wouldn't add it to the `out`.